docs/README.md

```diff
@@ -69,6 +69,20 @@ Route::get('/demo', function () {
 ```
 
 ## Upgrading
+### v1.6.0
+This release fixes accessibility concerns
+
+There are no breaking changes in this release.
+
+To get these changes on existing apps add the following to the \sass\nu_purple_layout.scss in your app.
```

Comment:
```suggestion
To get these changes on existing apps add the following to the `sass/nu_purple_layout.scss` in your app.
```

![imgflip-is-dead_windows-unix](https://user-images.githubusercontent.com/27235866/141493086-29482d62-0801-4769-8d22-b9d9822838e9.jpeg)